ebpf/xdp_filter.c

```diff
@@ -55,7 +73,8 @@ struct flowv4_keys {
         __u16 port16[2];
     };
     __u32 ip_proto;
-} __attribute__((__aligned__(8)));
+    __u16 vlan_id[2];
```

Comment:
does this contain just the vlan_id? The id is 12bit, so then we'd have 4 bits left and again for the 2nd vlan_id. We just support TCP or UDP, so instead of using 4 bytes for the ipproto, we could use 1 bit of one of the vlan_id fields.

I think this could be beneficial as we seem to be starved for space a bit. IIRC we can only use 64 bytes for key + value.